### ISSUE TITLE
i deleted twitch

### ISSUE THREAD

**Issue by cesarcaio** [mkvd/smarttv-twitch#69]
Hello , I have a Samsung Smart model J6400 i did the indicated with USB way and everything worked out correctly, but by accident i deleted the twitch TV application and now can install again. Already reseted TV in every way , I returned to factory settings but nothing happens when I put the stickusb again with the file. Is there any solution to this problem ?

**Comment by svetty** [comment]
did you remember to put it inside a folder called userwidget ?

**Comment by CenaJohn** [comment]
Hi, i  have the same issue with my ue40j6250 lastest firmware.
I even tried to install different apps via userwidget-folder but my tv just doenst react.
I get the message that my usb device is connected and if i would like to look through my files but nothing else.
Has anyone an idea how to fix this?

**Comment by svetty** [comment]
are you sure you are connecting the usb to the correct port ? My TV has 3 USB ports, while 2 of them are ONLY power.

**Comment by CenaJohn** [comment]
I dont think that really matters. I talked to samsung support during remote maintenance and i was told that they removed the "userwidget-folder" feature in 1422 cause it wasnt intented to be used by normal customers.
He said that i should try using the tizen-sdk and developer mode to emulate/sync the app file.
I tried that but i m too stupid to make it work.
Here is the manual:
http://stackoverflow.com/questions/34268755/tizen-app-not-installable-on-tv-firmware-1422
http://www.samsungdforum.com/TizenGuide/tizen3511/index.html
pls halp

**Comment by hendrikt1** [comment]
i have also deleted twitch.

my solution:
download samung TV Cast on your TV and on your mobile or tablet
look for your IP adres of your fone or tablet and type that in jour app. it will wireless connect to your TV. 
its like the same thing as chromecast. your fone or tablet is like a remote.

sorry for the bad english ^^

**Comment by CenaJohn** [comment]
Found a way to downgrade and then upgrade to the desired fw. Havent tried it out yet.
Here is the Link: http://forum.samygo.tv/viewtopic.php?f=87&t=10138
Mayby i ll try it when i have the time required
